simplq/src/components/common/ClickableLogo/clickableLogo.module.scss

```diff
@@ -3,6 +3,7 @@
 .logo {
   display: flex;
   cursor: pointer;
+  margin: 0 auto;
```

Comment:
Didn't really understand this change? This would push out the other links and the logo will move to the center, is this what you had intented?

![image](https://user-images.githubusercontent.com/11256376/104498712-12c8b980-5602-11eb-8ecd-afe90b06703d.png)

Reply:
yes

Reply:
What should be the change?

Reply:
Oh sorry, the issue was talking about this part: 
![image](https://user-images.githubusercontent.com/11256376/104503099-9f757680-5606-11eb-8d35-462d0be4d72e.png)

It looks fixed in the current code. Shall I close #408? What do you think?

Reply:
yes close it

Reply:
Actually the title and the "Ready to share" text below should be left aligned, but it's not, on smaller titles:

![image](https://user-images.githubusercontent.com/11256376/104503330-ea8f8980-5606-11eb-8a8c-2e26d98cbdcf.png)

Can you take a look?

Reply:
ok